### PR TITLE
New Wazuh API documentation

### PR DESCRIPTION
**Pull request by davidjiglesias**
Hello team,

This PR is related to https://github.com/wazuh/wazuh-documentation/issues/2460

This PR adds changes to the documentation for the new Wazuh API.

Best regards,

David J. Iglesias

### CODE REVIEW

**Review by CarlosRS9 — APPROVED** [review]

File: source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_packages_amazon.rst [hunk 1]
```diff
@@ -88,11 +71,11 @@ Installing the Wazuh API
       # service wazuh-api status
```
Comment (alberpilot) [inline source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_packages_amazon.rst:71]: With the new packages changes, the status must be checked with wazuh-manager status.

File: source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_packages_amazon.rst [hunk 2]
```diff
@@ -88,11 +71,11 @@ Installing the Wazuh API
...
 
 .. note::
-    Now that the Wazuh API is installed, check out the section :ref:`securing_api` to set up some additional settings.
+    Check out the section :doc:`RESTful API <../../../../user-manual/api/index>` for more information on how to set up and use Wazuh API.
```
Comment (alberpilot) [inline source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_packages_amazon.rst:74]: Please use `:ref:` instead `:doc:`. If a path change occurs, this link will be broken.

File: source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_sources_amazon.rst [hunk 3]
```diff
@@ -92,24 +92,13 @@ Installing Wazuh manager
...
+.. note:: You can also run an :ref:`unattended installation <unattended-installation>` for the Wazuh manager and API.
 
-  .. code-block:: console
+.. versionadded:: 4.0.0
 
-      # curl -s -o install_api.sh https://raw.githubusercontent.com/wazuh/wazuh-api/v|WAZUH_LATEST|/install_api.sh && bash ./install_api.sh download
+The Wazuh API will be installed along the Wazuh manager by default. No extra steps or requirements are needed to install it.
 
-3. Once the process is complete, you can check the service status with:
+To check the API service status use the following command:
```
Comment (alberpilot) [inline source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_sources_amazon.rst:101]: Please, apply the Wazuh manager status review in all documents.

File: source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_sources_amazon.rst [hunk 4]
```diff
@@ -123,7 +112,7 @@ Installing Wazuh API
 
       # service wazuh-api status
 
-.. note:: You can also run an :ref:`unattended installation <unattended-installation>` for the Wazuh manager and API.
+.. note:: Check out the section :doc:`RESTful API <../../../../user-manual/api/index>` for more information on how to set up and use Wazuh API.
```
Comment (alberpilot) [inline source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_sources_amazon.rst:115]: Same here, please apply the `ref` changes in all documents.

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: source/_static/js/redirects.js [hunk 5]
```diff
@@ -34,8 +34,23 @@ removedUrls['x.y'] = [
 ];
 */
 
+/* *** RELEASE 4.0 ****/
+
+/* Pages added in 4.0 */
+
+newUrls['4.0'] = [
```
[the remark this answers is not in the document]
Comment (davidjiglesias) [reply source/_static/js/redirects.js:41]: Thx!

File: source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_packages_amazon.rst (hunk 1 above)
Comment (davidjiglesias) [reply source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_packages_amazon.rst:71]: To be changed in corresponding issue https://github.com/wazuh/wazuh/issues/5824

File: source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_sources_amazon.rst (hunk 3 above)
Comment (davidjiglesias) [reply source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_sources_amazon.rst:101]: To be changed in corresponding issue wazuh/wazuh#5824

File: source/user-manual/api/getting-started.rst [hunk 6]
```diff
@@ -24,78 +24,341 @@ The API starts at boot time. To control or check the **wazuh-api** service, use
...
 
-Use the cURL command to send a *request* to confirm that everything is working as expected:
+.. _api_log_in:
+
+Logging into the API
+--------------------
+
+Wazuh API endpoints require authentication in order to be used. Therefore, all calls must include a JSON Web Token. JWT is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object. Follow the steps below to log in and obtain a token in order to run any endpoint:
+
+1. Use the cURL command to log in, the API will provide a JWT token upon success. Replace <user> and <password> with yours. By default, the user is ``wazuh`` and the password is ``wazuh``. If ``SSL`` (https) is enabled in the API and it is using the default **self-signed certificates**, it will be necessary to add the parameter ``-k``. Use the ``raw`` option to get the token in a plain text format. Querying the login endpoint with ``raw=true`` is highly recommended when using cURL commands as tokens could be really long and difficult to handle otherwise. Exporting the token to an environment variable will ease the use of API requests after login.
```
Comment (davidfmiranda) [inline source/user-manual/api/getting-started.rst:34]: Replace the enumerations by `#.`

**Review by alberpilot — APPROVED** [review]
LGTM

**Review by davidfmiranda — CHANGES_REQUESTED** [review]
Hello @davidjiglesias, I've reviewed your PR and LGTM, nevertheless, I've made a couple of suggestions. The first one is to omit the contracted forms such as `it's` or `you're`.

The second one is to replace all the numbers in enumerations by `#.` to make the maintenance easier.

Regards,

David

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**